src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user in alphabetical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists all persons in the address book to the user in alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<Person> allPersons = addressBook.getAllPersons().mutableListView();
+        Comparator<Person> byName = Comparator.comparing(Person::getName);
+        Collections.sort(allPersons, byName);
```

Comment:
Would be good to encapsulate the sorting function within the `AddressBook`, much like how `addPerson` and `removePerson` are `AddressBook` methods instead of being defined by the commands. This breaks the abstraction barrier.